examples/walkthrough/main.go

```diff
@@ -13,6 +13,7 @@ import (
 	"log"
 	"os"
 
+	"github.com/joho/godotenv"
```

Comment:
We probably don't want extra dependencies just to run the example. Does this extra code end up in the compiled library libuplink.so?

Reply:
> This is not a fix, this is an extra feature to put in the example.
> 
> You can run the example without a .env file like so:
> 
> ```
> ACCESS_GRANT=1cCYAHeo6b3QziQe9B9VXKtgBUJ6D6ZbAovEvr2U4DqND9iGTmqrX37NQcugmjDmsgKHs6XbqRyr3RV2RKjFPSrYeQfqJT3mNB8SisU2WxrvyuvN6bRESbWtsEfpKNaKhBMcrf93oTKo4eNP5GLrJzZ3nRrBq7bH7c7msAtefjApACNR9NYJFtoLyDUeaXXTBZAwTFWkNxCi6YBeJwRMFRyPmNcr1tcBF9hp8YntReoV1Su8kjA8fp1z1wAqbGVzLGrD8xBjX3DH go run ./exmamples/walkthrough
> ```
> 
> (this gives another error about the user agent but you can observe that the access grant is parsed and accepted)

Alright, that is clear to me now, and no the code is just for the walkthrough.